src/Illuminate/Cache/RateLimiting/Limit.php

```diff
@@ -35,16 +35,15 @@ class Limit
     /**
      * Create a new limit instance.
      *
-     * @param  mixed|string  $key
-     * @param  int  $maxAttempts
-     * @param  int  $decayMinutes
-     * @return void
+     * @param mixed|string $key
+     * @param int $maxAttempts
+     * @param int $decaySeconds
      */
-    public function __construct($key = '', int $maxAttempts = 60, int $decayMinutes = 1)
+    public function __construct($key = '', int $maxAttempts = 60, int $decaySeconds = 1)
```

Comment:
That is a major breaking change. Better to change this to accept a float instead, or you will have to target master.

Reply:
Great idea, I'll rework this to just accept fractions